src/main/java/salesforce/ui/pages/campaignlist/CampaignListLightningPage.java

```diff
@@ -51,9 +52,9 @@ public AbstractNewCampaignPage clickOnNewButton() {
 
     @Override
     public AbstractCampaignPage selectCampaignName(final String campaignName) {
-        String opportunityNameCss = String.format(NAME_CAMPAIGN, campaignName);
+        String opportunityNameXpath = String.format(NAME_CAMPAIGN, campaignName);
         nameCampaignSelected = WebDriverManager.getInstance().getWebDriver().findElement(By
-                .cssSelector(opportunityNameCss));
+                .xpath(opportunityNameXpath));
```

Comment:
if it is posible this line can be together with previous line

Reply:
It is not possible